active/NumberBox/NumberBox.md

```diff
@@ -156,55 +129,49 @@ unsealed runtimeclass NumberBox : Windows.UI.Xaml.Controls.Control
     Double StepFrequency;
 
     String Header;
+    DataTemplate HeaderTemplate;
     String Text; 
     String PlaceholderText;
-
-    NumberBoxBasicValidationMode BasicValidationMode;
+  
+    NumberBoxValidationMode ValidationMode;
 
     Boolean AcceptsCalculation;
 
     NumberBoxSpinButtonPlacementMode SpinButtonPlacementMode{ get; set; };
```

Comment:
Now that the SpinButton is a Flyout that will appear above the control, I believe this should be renamed to "IsSpinButtonEnabled".  

@MikeHillberg, @teaP, does that sound ok to you?

Reply:
I take that back - I am paging in our extensibility conversation. An enum makes it easy to add more placement modes later.

Reply:
Oh, uh, I was going to leave them both in. So there would be three modes -- none, inline, and flyout. Does no one want inline as an option?

Reply:
I forget you had both already prototyped. This was just to save cost on secondary configurations. If the implementation work is done, let's keep them both! 

I'll fix the doc and API to acknowledge this.